ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -579,6 +596,7 @@ PrjFS_Result PrjFS_DeleteFile(
     {
         // Only files and directories can be deleted with PrjFS_DeleteFile
         // Anything else should be treated as a full file
+        LogWarning("PrjFS_DeleteFile: failing due to full file fullPath=%s", fullPath);
```

Comment:
- Also log the values of `S_ISREG` and  `S_ISDIR`
- The message `"due to full file"` is a bit misleading, this check is unrelated to whether the file is full, it's checking if the path is a regular file or directory.